versions.properties

```diff
@@ -3,46 +3,47 @@
 ## Dependencies and Plugin versions with their available updates
 ## Generated by $ ./gradlew refreshVersions
 ## Please, don't put extra comments in that file yet, keeping them is not supported yet.
-plugin.android=4.0.0
+plugin.android=4.0.1
 version.androidx.activity=1.1.0
 version.androidx.appcompat=1.1.0
-version.androidx.constraintlayout=1.1.3
+version.androidx.constraintlayout=2.0.1
 version.androidx.databinding=4.0.0
 version.androidx.fragment=1.2.5
 version.androidx.gridlayout=1.0.0
 version.androidx.lifecycle=2.2.0
 version.androidx.recyclerview=1.1.0
 version.androidx.savedstate=1.0.0
-version.androidx.test.espresso=3.2.0
-version.androidx.test.ext.junit=1.1.1
+version.androidx.test.espresso=3.3.0
+version.androidx.test.ext.junit=1.1.2
 version.androidx.test.uiautomator=2.2.0
-version.androidx.test=1.2.0
+version.androidx.test=1.3.0
 version.androidx.transition=1.3.1
 version.com.android.tools..desugar_jdk_libs=1.0.9
+version.com.github.ben-manes..gradle-versions-plugin=0.33.0
 version.com.google.truth..truth=1.0.1
 version.com.googlecode.lanterna..lanterna=3.0.3
 version.com.jakewharton.timber..timber=4.7.1
 version.com.nhaarman..mockito-kotlin-kt1.1=1.6.0
 version.com.squareup..seismic=1.0.2
-version.com.vanniktech..gradle-maven-publish-plugin=0.12.0
+version.com.vanniktech..gradle-maven-publish-plugin=0.13.0
 version.cycler=0.1.4
 version.google.android.material=1.1.0
 version.io.gitlab.arturbosch.detekt..detekt-gradle-plugin=1.0.1
 version.io.mockk..mockk=1.10.0
 version.io.reactivex.rxjava2..rxandroid=2.1.1
 version.io.reactivex.rxjava2..rxjava=2.2.19
 version.junit.junit=4.13
-version.kotlin=1.3.72
-version.kotlinx.coroutines=1.3.7
-version.kotlinx.serialization=0.20.0
-version.me.champeau.gradle..jmh-gradle-plugin=0.5.0
-version.moshi=1.9.3
+version.kotlin=1.4.10
+version.kotlinx.coroutines=1.3.9
```

Comment:
Can't upgrade coroutines yet because we can't require the 1.4 stdlib.

Reply:
☹️